data/config/default.json

```diff
@@ -254,15 +254,48 @@
       },
       {
         "enabled": true,
-        "type": "export-to-workspace",
-        "title": "Export To Workspace",
+        "type": "manifest",
+        "title": "Download Manifest",
         "leftIcon": "datafile",
         "rightIcon": "download",
         "fileName": "manifest.json"
       }
     ],
     "table": {
-      "enabled": true
+      "enabled": true,
+      "fields": [
+        "project",
+        "study",
+        "race",
+        "ethnicity",
+        "gender",
+        "vital_status",
+        "whatever_lab_result_value",
+        "file_count",
+        "file_type",
+        "file_format"
+      ]
+    },
+    "guppyConfig": {
+      "dataType": "subject",
+      "fieldMapping": [
+        { "field": "project", "name": "Project" },
```

Comment:
Should we automate this part to only configure the special mapping?
I mean if it is only `project -> Project` or `vital_status -> Vital Status` you don't need to put it in the fieldMapping.

Reply:
I think sometimes some field name mapping will need more caring, like `thrpyv`, but yeah automation is a good idea, I think we could combine both automation and customized names